### PR TITLE
fix: adjust dashboard widget height

### PR DESCRIPTION
**Pull request by ccailly**
### Checklist

Please check if your PR fulfills the following specifications:

- [ ] Tests for the changes have been added
- [ ] Docs have been added/updated

### References

Closes #N/A

Correct dashboard widget height in `install.php`.
Previous pull request, #3306 refactored UI of widgets but missed updating install.php.
A subsequent commit updated `install.php` but set widget height to 1 instead of the correct value 2.

This PR fixes the widget height value to 2.

### CODE REVIEW

File: install/upgrade_to_2.13.10.php [hunk 1]
```diff
@@ -0,0 +1,79 @@
+<?php
```
Comment (Rom1-B) [inline install/upgrade_to_2.13.10.php:1]: It seems simpler and less risky not to resize at all at the update, neither in this version nor in 2.13.7.

**Review by Rom1-B — CHANGES_REQUESTED** [review]

**Review by btry — CHANGES_REQUESTED** [review]
To upgrade to version 2.13.0 the version constant in setup.php must be up to date. Also in install.php there is a matrix which modelizes versions sequence It needs a new entry 2.13.9 => 2.13.10.

Also, I don't expect to release a new version as the plugin reached it end of life and we are implementing forms in GLPI core (v11).

**Review by Rom1-B — APPROVED** [review]

**Comment by ccailly** [comment]
As discussed with @Rom1-B above and IRL, it's probably wiser not to change the widget size during an update.
It's possible that some users may have manually modified their widgets, so only new installations of the plugin will benefit from the new widget size.

**Comment by btry** [comment]
Hi 

> As discussed with @Rom1-B above and IRL, it's probably wiser not to change the widget size during an update.
It's possible that some users may have manually modified their widgets, so only new installations of the plugin will benefit from the new widget size.

Actually, the plugin takes measures to prevent users to modify its mini dashboard. Noone should be able to modify it (neither super-admins). This is why it should be safe to modify it programmatically

**Review by AdrienClairembault — CHANGES_REQUESTED** [review]
Agreed with @btry.

The `resizeWidgets` function must not be deleted.

**Review by AdrienClairembault — APPROVED** [review]

**Review by btry — APPROVED** [review]

**Pull request merged**